src/app/views/app_frame/frame/main/views/dashboard/timeline/index.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+
+import {Row, Component} from '../../../components/layouts'
+import Timeline from '@material-ui/lab/Timeline';
```

Comment:
Dont worry about it now but in the future:
`import { Foo, Bar, Baz } from '@material-ui/lab`

Reply:
also no semicolon